### PR TITLE
remove dependency on marionette

### PR DESCRIPTION
**Pull request by samandmoore**
knocking this out: https://github.com/Betterment/backbone.blazer/issues/6

@dschaub 

/cc @jmeas

### CODE REVIEW

File: backbone.blazer.js [hunk 1]
```diff
@@ -2,22 +2,34 @@
...
+
+    Backbone.Blazer.Route.extend = Backbone.Model.extend;
+
+    _.extend(Backbone.Blazer.Route.prototype, Backbone.Events, {
+        initialize: function() {},
+        destroy: function() {
+            this.stopListening();
+            return this;
+        },
         prepare: function() { return $.Deferred().resolve(); },
```
Comment (dschaub) [inline backbone.blazer.js:31]: can you throw a `.promise()` on the end of that?

Comment (jamesplease) [inline backbone.blazer.js:31]: This seems out of scope for this PR?

Comment (dschaub) [inline backbone.blazer.js:31]: yup, my bad

Comment (samandmoore) [inline backbone.blazer.js:31]: eager beaver :traffic_light:
